web/src/popovers.js

```diff
@@ -103,7 +103,10 @@ export function popover_items_handle_keyboard(key, $items) {
         return;
     }
     if (index === -1) {
-        if ($(".user_info_popover_manage_menu_btn").is(":visible")) {
+        if (
+            $(".user_info_popover_manage_menu_btn").is(":visible") &&
+            !user_card_popover.user_info_manage_menu_popped()
+        ) {
```

Comment:
I don't understand what this code is about at all -- it's not obvious from looking at this function why the user card popover should be special at all. Can you add some comments that provide that background?

Reply:
Pushed commit da383f9 to fix this. Hope there's enough context.